### ISSUE TITLE
Getting the dataset

### ISSUE THREAD

**Issue by Livvi** [yahoo/FEL#21]
Hi,

I would like to use the entity linker and I am trying to get the datasets from the links given in the readme file.

If I log in with my yahoo account and try to submit a request to get the L30 dataset, nothing happens. I am redirected to the "My dataset selection" page again, the request does not seem to be send. 

How can I get the dataset?

**Comment by lovejing0306** [comment]
Have you the dataset?

**Comment by aasish** [comment]
Hello, 

This employee can no longer access email on this account.  Your email will not be forwarded.

If you have a sales inquiry, please contact sales@yahoo-inc.com

If you require assistance with a legal matter, please contact legal-notices@yahoo-inc.com

Thank you!

**Comment by Livvi** [comment]
No, I never received any answer and I don't have the dataset. It seems like this project is no longer maintained.

**Comment by lovejing0306** [comment]
I think so, I try to download the dataset but failure.

**Comment by huaweiDu** [comment]
> I think so, I try to download the dataset but failure.

hello, have you have the dataset?

Can you share one with me? 

Thank you！

**Comment by aasish** [comment]
Hello, 

This employee can no longer access email on this account.  Your email will not be forwarded.

If you have a sales inquiry, please contact sales@yahoo-inc.com

If you require assistance with a legal matter, please contact legal-notices@yahoo-inc.com

Thank you!
